packages/components/Molecules/Card/__test__/CardNav.test.js

```diff
@@ -45,7 +45,7 @@ describe('CardNav', () => {
     expect(wrapper.vm.trimmedSummary).toHaveLength(200 + 3)
   })
 
-  it('formats the date correctly', () => {
+  it('formats the date correctly', async () => {
```

Comment:
I'm curious why changing these tests to be async was needed ? Is it related to the change from moment to dayJs?

Reply:
@dylankelly, After dayjs library was installed SetProps was not working as expected. After some findings I got the conclusion that setProps won't work when installing library in the project.
Javascript is synchronous by default and it will continuously run without waiting. So adding async() await will be the solution.